scala/ppml/demo/README.md

```diff
@@ -1,36 +1,63 @@
-# PPML Demo
+# Run PPML Demo in Graphene
 
-## Get Prepared
-### Get jar ready
-#### Build from source
+## Before running code
+
```

Comment:
Add https://github.com/intel-analytics/BigDL/tree/branch-2.0/ppml/trusted-big-data-ml/scala/docker-occlum#prerequisites for Graphene part